flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -86,40 +104,93 @@ class FlxGamepad implements IFlxDestroyable
 	private var _isChrome:Bool = false;
 	#end
 	
-	public function new(ID:Int, Manager:FlxGamepadManager, ?Model:FlxGamepadModel) 
+	public function new(ID:Int, Manager:FlxGamepadManager, ?Model:FlxGamepadModel, ?Attachment:FlxGamepadModelAttachment) 
 	{
 		id = ID;
 		
 		manager = Manager;
 		
+		pressed = new FlxGamepadButtonList(FlxInputState.PRESSED, this);
+		justPressed = new FlxGamepadButtonList(FlxInputState.JUST_PRESSED, this);
+		justReleased = new FlxGamepadButtonList(FlxInputState.JUST_RELEASED, this);
+		analog = new FlxGamepadAnalogList(this);
+		motion = new FlxGamepadMotionValueList(this);
+		pointer = new FlxGamepadPointerValueList(this);
+		
 		if (Model == null)
 			Model = XBox360;
+			
+		if (Attachment == null)
+			Attachment = None;
 		
-		buttonIndex = new FlxGamepadMapping(model);
+		buttonIndex = new FlxGamepadMapping(model, attachment);
 		model = Model;
 		detectedModel = Model;
 		
 		#if flash
 		_isChrome = (Capabilities.manufacturer == "Google Pepper");
 		#end
-		
-		pressed = new FlxGamepadButtonList(FlxInputState.PRESSED, this);
-		justPressed = new FlxGamepadButtonList(FlxInputState.JUST_PRESSED, this);
-		justReleased = new FlxGamepadButtonList(FlxInputState.JUST_RELEASED, this);
-		analog = new FlxGamepadAnalogList(this);
+	}
+	
+	public function traceAxes(f:FlxTimer):Void
```

Comment:
they are, but could be useful to keep around in the API if they're not actually called... maybe? I use them all the time.